docs/device_automation_trigger.md

```diff
@@ -99,3 +100,18 @@ async def async_attach_trigger(hass, config, action, automation_info):
 ```
 
 The return value is a function that detaches the trigger.
+
+#### Add text and translations
+
+The Automation user interface will display a human-readable string in the device automation mapped to the event
+type.  Update `strings.json` and if you used `script.scaffold` then `translations/en.json` with the full set of supported event types:
+
+```json
+{
+   "device_automation": {
+    "trigger_type": {
+      "water_detected": "Water detected",
+      "noise_detected": "Noise detected"
+    }
+}
```

Comment:
```suggestion
type.  Update `strings.json` with the trigger types and subtypes that you support:

```json
{
   "device_automation": {
    "trigger_type": {
      "remote_button_short_press": "\"{subtype}\" button pressed",
      "remote_button_short_release": "\"{subtype}\" button released"
    },
    "trigger_subtype": {
      "turn_on": "Turn on",
      "turn_off": "Turn off"
    }
}
\```

To test your translations during development, run `python3 -m script.translations develop`.

```

Reply:
Comment failure because of the JSON block being inside the suggestion